taxi/forms.py

```diff
@@ -0,0 +1,65 @@
+from django.contrib.auth import get_user_model
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Car
+
+
+class DriverLicenseValidator:
+    COUNT_LETTERS = 3
+    COUNT_NUMBERS = 5
+    LICENSE_NUMBER_LENGTH = COUNT_LETTERS + COUNT_NUMBERS

```

Comment:
It's not really number length. I think, you should rename this constant

Reply:
yep)